.github/files/build-all-projects.sh

```diff
@@ -141,6 +141,22 @@ for SLUG in "${SLUGS[@]}"; do
 		cp -r "$BASE/.github/files/gh-autotagger/." "$BUILD_DIR/.github/."
 	fi
 
+	# Copy license.
+	LICENSE=$(jq -r '.license // ""' composer.json)
+	if [[ "$LICENSE" ]]; then
```

Comment:
I like to be a little more explicit in these tests. I think bash is smart enough to not break even if someone sets the license to something that looks like an operator, but it's nice to not have to worry about it.
```suggestion
	if [[ -n "$LICENSE" ]]; then
```